src/popup/router/components/Modals/AccountDetails.vue

```diff
@@ -0,0 +1,332 @@
+<template>
+  <Modal class="account-details">
+    <div ref="account_details">
+      <div class="account-info-wrapper">
+        <AccountInfo
+          v-bind="$attrs"
+          color="#212121"
+          show-copy-icon
+        />
+        <ButtonPlain @click="resolve">
+          <Close />
+        </ButtonPlain>
+      </div>
+      <BalanceInfo v-bind="$attrs" />
+      <div class="buttons">
+        <ButtonPlain>
+          <ArrowReceive />
+          <div>{{ $t('pages.token-details.receive') }}</div>
+        </ButtonPlain>
+        <ButtonPlain>
+          <ArrowSend />
+          <div>{{ $t('pages.token-details.send') }}</div>
+        </ButtonPlain>
+        <ButtonPlain>
+          <CreditCard />
+          <div>{{ $t('pages.token-details.buy') }}</div>
+        </ButtonPlain>
+        <ButtonPlain>
+          <Swap />
+          <div>{{ $t('pages.token-details.swap') }}</div>
+        </ButtonPlain>
+      </div>
+      <div class="tabs-wrapper">
+        <div class="tabs">
+          <ButtonPlain
+            :class="{active: activeTab === 'tokens'}"
+            @click.prevent="activeTab = 'tokens'"
+          >
+            {{ $t('pages.fungible-tokens.tokens') }}
+          </ButtonPlain>
+          <ButtonPlain
+            :class="{active: activeTab === 'transactions'}"
+            @click.prevent="activeTab = 'transactions'"
+          >
+            {{ $t('pages.token-details.transactions') }}
+          </ButtonPlain>
+          <ButtonPlain
+            :class="{active: activeTab === 'names'}"
+            @click.prevent="activeTab = 'names'"
+          >
+            {{ $t('pages.titles.names') }}
+          </ButtonPlain>
```

Comment:
To not repeat the code I'd suggest to make an array with all the tab names
```
tabs: ['tokens', 'transactions', 'names'],
```

and then iterate through them with v-for

```
<ButtonPlain
            v-for="tab in tabs"
            :key="tab"
            :class="{ active: activeTab === tab }"
            @click.prevent="setActiveTab(tab)"
          >
            {{ $t(`pages.account-details.${tab}') }}
          </ButtonPlain>
```

Of course translation strings should be adjusted accordingly